### DISCUSSION TITLE
DSA Basket

### DISCUSSION THREAD

**Discussion by rahulsrivastava1** [scaleracademy/open-source-projects#92] [Ideas]
Everyone knows for better placement we have to learn DSA very well and grasp almost all the algorithm with lots of practice that will help anyone to crack the FAANG and many other multinational product based companies.
There are lots of DSA sheet provided by our top Youtubers and we can also share the solutions of different sheets in different languages so that if anyone wanted any problem's solution in any language then, he/she should find their solution from this repository.

So, that's all I have to say.

Thank you!

**Comment by iamrahul8** [comment]
Yupp🙌
DSA/programming question solution with explanation will be a good idea

**Reply by rahulsrivastava1** [reply]
Thank you @iamrahul8

**Comment by 21pravi** [comment]
great idea
